frontend/src/components/ChatResponse.module.css

```diff
@@ -178,29 +178,9 @@
     padding-left: 0.25rem;
 }
 
-.li > .ul,
-.li > .ol {
-    margin: 0 rem 0;
-}
-
-.aiContent .li > .p,
-.toolContent .li > .p {
-    margin: 0;
-}
-
-.aiContent > .p,
-.toolContent > .p {
-    margin: 0.5rem 0;
-}
-
-.aiContent > .p:first-child,
-.toolContent > .p:first-child {
-    margin-top: 0;
-}
-
-.aiContent > .p:last-child,
-.toolContent > .p:last-child {
-    margin-bottom: 0;
+.ul > .li > .p,
+.ol > .li > .p {
+    margin-top: calc(-1.6em);
```

Comment:
- calc() converts em to px
- 1.6 is the `line-height` used throughout
- `-1.6` to offset